src/atomate2/vasp/flows/mpmorph.py

```diff
@@ -0,0 +1,539 @@
+"""Define VASP implementations of MPMorph flows.
+
+For information about the current flows, contact:
+- Bryant Li (@BryantLi-BLI)
+- Aaron Kaplan (@esoteric-ephemera)
+- Max Gallant (@mcgalcode)
+"""
+
+from __future__ import annotations
+
+import math
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from jobflow import Response
+
+from atomate2.common.flows.mpmorph import (
+    EquilibriumVolumeMaker,
+    FastQuenchMaker,
+    MPMorphMDMaker,
+    SlowQuenchMaker,
+)
+from atomate2.vasp.flows.md import MultiMDMaker
+from atomate2.vasp.jobs.mp import (
+    MPMetaGGARelaxMaker,
+    MPMetaGGAStaticMaker,
+    MPPreRelaxMaker,
+)
+from atomate2.vasp.jobs.mpmorph import (
+    BaseMPMorphMDMaker,
+    FastQuenchVaspMaker,
+    SlowQuenchVaspMaker,
+)
+from atomate2.vasp.powerups import update_user_incar_settings
+
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Any
+
+    from jobflow import Flow
+    from pymatgen.core import Structure
+
+    from atomate2.vasp.jobs.md import MDMaker
+
+
+@dataclass
+class BaseMPMorphVaspMDMaker(MPMorphMDMaker):
+    """Base MPMorph flow for amorphous solid equilibration using VASP.
+
+    This flow uses NVT molecular dynamics to:
+    (1 - optional) Determine the equilibrium volume of an amorphous
+        structure via EOS fit.
+    (2 - optional) Quench the equilibrium volume structure from a higher
+        temperature down to a lower desired "production" temperature.
+    (3) Run a production, longer-time MD run in NVT.
+        The production run can be broken up into smaller steps to
+        ensure the simulation does not hit wall time limits.
+
+    Check atomate2.common.flows.mpmorph for MPMorphMDMaker
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    temperature : float = 300
+        Temperature of the equilibrium volume search and production run in Kelvin,
+        default 300K
+    end_temp : float = 300
+        End temperature of the equilibrium volume search and production run in Kelvin,
+        default 300K.
+        Use only for lowering temperarture for production run
+    md_maker : BaseMPMorphMDMaker
+        MDMaker to generate the molecular dynamics jobs specifically for MPMorph AIMD;
+        inherits from MDMaker (VASP)
+    steps_convergence: int | None = None
+        Defaults to 5000 steps unless specified
+    steps_single_production_run: int | None = 5000
+        Number of steps for a single production run; default 5000 steps.
+        If set and steps_total_production > steps_single_production_run,
+        multiple production runs (MultiMDMaker) will be generated.
```

Comment:
My comments on the force field maker re `quench_maker_kwargs`, setting `Maker` parameters in the `args`, and the need for a `_post_init_kwargs` argument also apply here.

Reply:
Cleaned up with previous comments, I think